db/test/test_data.py

```diff
@@ -320,6 +320,51 @@ def test_load_high_level_missing_offset(self):
 
         self.assertDictEqual(hl3_expected, db.data.load_high_level(self.test_mbid, offset=2))
 
+    
+    def test_map_class_labels(self):
+        """Mapping of shortened keys in model classes of high-level data"""
+
+        hl = {"highlevel": {"genre_tzanetakis": {"all": {
+            "blu":0,
+            "cla":0,
+            "cou":0,
+            "dis":0,
+            "hip":0,
+            "jaz":0,
+            "met":0,
+            "pop":0,
+            "reg":0,
+            "roc":0 },
+            "value": " "}}}
+
+        hl_expected = {"highlevel": {"genre_tzanetakis": {"all": {
+            "blues":0,
+            "classical":0,
+            "country":0,
+            "disco":0,
+            "hiphop":0,
+            "jazz":0,
+            "metal":0,
+            "pop":0,
+            "reggae":0,
+            "rock":0 },
+            "value":" "}}}
+
+        db.data.add_model('genre_tzanetakis', 'v1', 'show')
+        db.data.set_model_mappings('genre_tzanetakis', 'v1', """{
```

Comment:
You shouldn't need this, because tests that inherit from `DatabaseTestCase` will load the database schema and run the `create_models.sql` script. Did you find that this model didn't exist in the `models` table?

Reply:
I found that the mappings didn't exist when ran the test script, but now that I recall, I must have written this test before i moved the mappings to `create_models.sql`. And yes that add_model shouldn't be there, ill remove it